tests/util/blockdev_utils.rs

```diff
@@ -37,14 +43,30 @@ pub fn clean_blockdev_headers(blockdev_paths: &Vec<&Path>) {
     for path in blockdev_paths {
         match wipe_header(path) {
             Ok(_) => {}
-            Err(err) => {
-                panic!("Failed to clean signature on {:?} : {:?}", path, err);
+            Err(e) => {
+                panic!("Failed to clean signature on {:?} : {:?}", path, e);
             }
         }
     }
     info!("devices cleaned for test");
 }
 
+#[allow(dead_code)]
```

Comment:
It is not dead code, but if I remove the directive I get a compiler warning for dead code.

I'm not sure why.